docs/howto/release-git-lfs.md

```diff
@@ -117,11 +117,17 @@ to zero, we are releasing a PATCH version.
          `release-M.(N-1)`.)
 
      * Run `script/update-version vM.N.P` to update the version number in all
-       of the relevant files.
+       of the relevant files.  Note that this script requires a version of
+       `sed(1)` compatible with the GNU implementation.
```

Comment:
Just FYI, you could add a patch to make this use Ruby instead of documenting this behaviour.  I'm sure this is my fault, and we already use Ruby elsewhere in the developer tooling.

Reply:
I thought about that, but I confess I took the easy route here, because there are a number of other places where our release tooling more or less assumes the user is on a Linux system, and I figured that if I ever tackled those more comprehensively, I'd do so in a separate PR.